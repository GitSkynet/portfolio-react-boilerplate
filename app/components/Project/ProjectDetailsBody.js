import styled from 'styled-components';

const ProjectDetailsBody = styled.p`
  min-width: 32vw;
  margin-top: 0.5em;
`;

export default ProjectDetailsBody;
