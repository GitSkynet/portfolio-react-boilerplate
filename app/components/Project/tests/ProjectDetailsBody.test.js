import React from 'react';
import { shallow } from 'enzyme';

import ProjectDetailsBody from '../ProjectDetailsBody';

describe('<ProjectDetailsBody />', () => {
  it('should render and match the snapshot', () => {
    const wrapper = shallow(<ProjectDetailsBody />);
    expect(wrapper).toMatchSnapshot();
  });
});
