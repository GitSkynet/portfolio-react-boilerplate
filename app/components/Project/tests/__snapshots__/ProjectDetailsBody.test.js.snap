// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProjectDetailsBody /> should render and match the snapshot 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <p
    className="ProjectDetailsBody__ProjectDetailsBody-lGapx eBNymQ"
/>,
  "nodes": Array [
    <p
      className="ProjectDetailsBody__ProjectDetailsBody-lGapx eBNymQ"
/>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <ProjectDetailsBody__ProjectDetailsBody />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StyledComponent {
        "_reactInternalInstance": [Circular],
        "attrs": Object {},
        "context": Object {
          "__styled-components-stylesheet__": undefined,
          "__styled-components__": undefined,
        },
        "props": Object {},
        "refs": Object {},
        "state": Object {
          "generatedClassName": "eBNymQ",
          "theme": Object {},
        },
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <p
          className="ProjectDetailsBody__ProjectDetailsBody-lGapx eBNymQ"
/>,
        "_debugID": 2,
        "_renderedOutput": <p
          className="ProjectDetailsBody__ProjectDetailsBody-lGapx eBNymQ"
/>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <ProjectDetailsBody__ProjectDetailsBody />,
}
`;
